packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -805,6 +808,10 @@ export class LibpodDockerode {
         options: {},
       };
 
+      // if we don't build - we should send Content-Type application/yaml
```

Comment:
I could be remembering this incorrectly, but wasn't there a bug / PR pushed to correctly use `application/tar` or was that a different API call that was being updated upstream side?

Reply:
When adding the build support in the api (https://github.com/containers/podman/pull/24015) they changed the accepted content type.

- when sending json/yaml you need to use `application/yaml` or `application/json`.
- when sending tarball you need to use `application/x-tar`.

By default, dockerode uses `application/tar` but this is not supported anymore for kube play (see list of accepted content type in https://github.com/containers/podman/pull/24015/files#diff-de26209aed278aeaa5803ab09bcb66127ec5346f34cfa4e0b21fead87adc6afeR43-R53)

Reply:
Ah that makes sense, so much for clarifying. This LGTM.